code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -540,21 +565,68 @@
 
 	if(!istype(patient))
 		playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE)
-		to_chat(user, span_notice("\The [src] makes a sad buzz and briefly displays a frowny face, indicating it can't scan [patient]."))
+		to_chat(user, span_notice("\The [src] makes a sad buzz and briefly displays an unhappy face, indicating it can't scan [patient]."))
+		emotion = AID_EMOTION_SAD
+		update_appearance(UPDATE_OVERLAYS)
 		return
 
 	woundscan(user, patient, src)
+	update_appearance(UPDATE_OVERLAYS) // woundscan updates emotion
+	flick(icon_state + "_pinprick", src)
+
+/obj/item/healthanalyzer/wound/update_overlays()
+	. = ..()
+	switch(emotion)
+		if(AID_EMOTION_HAPPY)
+			. += mutable_appearance(icon, "+no_wounds")
+		if(AID_EMOTION_WARN)
+			. += mutable_appearance(icon, "+wound_warn")
+		if(AID_EMOTION_ANGRY)
+			. += mutable_appearance(icon, "+angry")
+		if(AID_EMOTION_SAD)
+			. += mutable_appearance(icon, "+fail_scan")
+	if(emotion != AID_EMOTION_NEUTRAL)
+		addtimer(CALLBACK(src, PROC_REF(reset_emotions)), 2 SECONDS)
+
+/obj/item/healthanalyzer/wound/proc/reset_emotions()
+	emotion = AID_EMOTION_NEUTRAL
+	update_appearance(UPDATE_OVERLAYS)
+
+/obj/item/healthanalyzer/wound/miner
+	name = "mining wound analyzer"
+	icon_state = "miner_aid"
+	desc = "A helpful, child-proofed, and most importantly, extremely cheap MeLo-Tech medical scanner used to diagnose injuries and recommend treatment for serious wounds. While it might not sound very informative for it to be able to tell you if you have a gaping hole in your body or not, it applies a temporary holoimage near the wound with information that is guaranteed to double the efficacy and speed of treatment. This one has a cool aesthetic antenna that doesn't actually do anything!"
 
 /obj/item/healthanalyzer/disease
 	name = "disease state analyzer"
-	icon_state = "adv_spectrometer"
-	desc = "A disease status analyzer to determine the state of an infection to encourage responsible medication consumption."
+	desc = "Another of MeLo-Tech's dubiously useful medsci scanners, the disease analyzer is a pretty rare find these days - NT found out that giving their hospitals the lowest-common-denominator pandemic equipment resulted in too much financial loss of life to be profitable. There's rumours that the inbuilt AI is jealous of the first aid analyzer's success."
+	icon_state = "disease_aid"
+	mode = SCANNER_NO_MODE
+	var/next_encouragement
+	var/emotion = AID_EMOTION_NEUTRAL
```

Comment:
I don't love how this is copy/pasted between different models, perhaps it could be put into its own datum with subtypes for the switch statement?

Reply:
gah